pkg/options/rawyamltmpl/raw_text_applier.go

```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package rawtexttmpl is a special-case component maker for constructing
-// objects with the assumption that objects have been inlined via RawText into
-// ConfigMaps.
+// Package rawtexttmpl is a special-case option applier for adding options to
+// objects with the assumption that objects have been inlined as go-templates
+// via RawTextFiles ComponentPackageSpec field into ConfigMaps.
```

Comment:
I think it's worth adding the 'Once the parameters are applied via the go template, ...' description to the package description as well, as a reader might very well look here first.

Reply:
Done